k8s/demo/common/money-claims/cmc-demo.yaml

```diff
@@ -56,7 +56,6 @@ spec:
         environment:
           FRONTEND_BASE_URL: "https://{{ .Values.global.cmcFrontendUrl }}"
           RESPOND_TO_CLAIM_URL: "https://{{ .Values.global.cmcFrontendUrl }}/first-contact/start"
-          FEATURE_TOGGLES_JDDO: true
```

Comment:
what this supposed to mean??

Reply:
This flag is removed from the application code and not needed anymore.